Index: app/src/main/java/com/example/pethub/MainActivity.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.pethub\r\n\r\nimport android.content.Intent\r\nimport androidx.appcompat.app.AppCompatActivity\r\nimport android.os.Bundle\r\nimport com.google.firebase.database.FirebaseDatabase\r\n\r\nclass MainActivity : AppCompatActivity() {\r\n    override fun onCreate(savedInstanceState: Bundle?) {\r\n        super.onCreate(savedInstanceState)\r\n        //setContentView(R.layout.activity_main)\r\n        //startActivity(Intent(this, Notification::class.java))\r\n\r\n        setContentView(R.layout.vet_profile)\r\n\r\n        // connect to firebase db\r\n        //var database = FirebaseDatabase.getInstance().reference\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/pethub/MainActivity.kt b/app/src/main/java/com/example/pethub/MainActivity.kt
--- a/app/src/main/java/com/example/pethub/MainActivity.kt	
+++ b/app/src/main/java/com/example/pethub/MainActivity.kt	
@@ -9,7 +9,7 @@
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         //setContentView(R.layout.activity_main)
-        //startActivity(Intent(this, Notification::class.java))
+        startActivity(Intent(this, VeterinaryClinic::class.java))
 
         setContentView(R.layout.vet_profile)
 
